app/Resources/views/base.html.twig

```diff
@@ -29,7 +29,7 @@
                     <li class="border-left">
                         <a href="{{ 'page.header.help_link'|trans }}" target="_blank">{{ 'page.header.help'|trans }}</a>
                     </li>
-                    <li class="border-left"><a>{{ 'page.header.logout'|trans }}</a></li>
+                    <li class="border-left"><a href="/logout">{{ 'page.header.logout'|trans }}</a></li>
```

Comment:
Why not use the `path` method here for the logout anchor?

Example:
`<a href="{{ path('entity_acl', {id: action.id, serviceId: action.serviceId}) }}">`

Reply:
What's the benefit of using it over a simple link?  It adds computation to delivering the html, when we will get the exact same link.  And it's not as if this link will change.  I assume I'm missing someting?

Reply:
Bigest gain is, when changing the destination of the route, you don't need to update your 'href' components. It also provides a very clean way to pass parameters and other route related modifiers.

Reply:
Not sure i'd agree that this case warrants it.  But adjusted to comply with best practice 👍